vault/router_test.go

```diff
@@ -373,6 +373,7 @@ func TestRouter_LoginPath(t *testing.T) {
 		{"auth/foo/bar", false},
 		{"auth/foo/login", true},
 		{"auth/foo/login/", false},
+		{"auth/invalid/login/", false},
```

Comment:
should this be `auth/invalid/login` , since `auth/foo/login/` is not a valid path, and so I'm guessing the trailing slash makes the login path invalid even if `invalid` were to be actually valid like `foo` ?